Dockerfile

```diff
@@ -0,0 +1,131 @@
+ARG BASE_IMAGE=nvcr.io/nvidia/l4t-pytorch:r32.4.3-pth1.6-py3
+FROM ${BASE_IMAGE}
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV JUPYTER_PASSWORD=jetbot
+
+WORKDIR /jetbot_dir
+
+ENV SHELL /bin/bash
+
+RUN apt-get update && apt-get install -y
```

Comment:
Do you know if this line does anything?  Thinking of removing in my branch

Reply:
We can remove this line.